utils/src/main/java/com/cloud/utils/nicira/nvp/plugin/NiciraNvpApiVersion.java

```diff
@@ -0,0 +1,60 @@
+//
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+//
+
+package com.cloud.utils.nicira.nvp.plugin;
+
+import org.apache.log4j.Logger;
+
+import com.cloud.maint.Version;
+
+public class NiciraNvpApiVersion {
+    private static final Logger s_logger = Logger.getLogger(NiciraNvpApiVersion.class);
+
+    private static String niciraApiVersion = null;
+
+    public static void pingNiciraApiVersion(String apiVersion){
```

Comment:
What is the whole point of this method? You already have a static method to set the version that accepts null. You also have another static method that checks if the a given API version is lower than the existing one.